src/Symfony/Component/HttpKernel/EventListener/ExceptionListener.php

```diff
@@ -65,7 +65,7 @@ public function onKernelException(GetResponseForExceptionEvent $event)
         $request->setMethod('GET');
 
         try {
-            $response = $event->getKernel()->handle($request, HttpKernelInterface::SUB_REQUEST, true);
+            $response = $event->getKernel()->handle($request, HttpKernelInterface::SUB_REQUEST, false);
```

Comment:
I think this change makes sense. But the `static $handling` variable seems obsolete now. It seems like it was introduced as a workaround for exactly that: https://github.com/nicolas-grekas/symfony/commit/71d5be1d6b239a87837a604b23308bce53fe99ff